### DISCUSSION TITLE
developing DApps

### DISCUSSION THREAD

**Discussion by drmadx** [IBAX-io/go-ibax#1752] [Ideas]
Making low-code development and zero-code coin issuance possible, which greatly reduces the time and barrier to developing DApps places IBAX above the contemporaries

**Comment by Bulent3407** [comment]
👍👍

**Reply by remaxx13** [reply]
Yes of course

**Reply by Sitoci** [reply]
yes you can

**Reply by DILJUS** [reply]
Yes

**Reply by aynurduzgun** [reply]
Evet elbette katılıyorum.umarim iyi bir kazanç getirecek.

**Reply by Tintnaingu** [reply]
Low code

**Reply by tuncuby** [reply]
best project

**Reply by loves96617665** [reply]
yes

**Comment by CODOMAN** [comment]
Beautiful beautiful wonderful 👍

**Reply by QuocPhu83** [reply]
Very very good

**Reply by trungcba** [reply]
really??

**Comment by berkpytr** [comment]
i want too

**Comment by Fermandar** [comment]
nice

**Reply by Sitoci** [reply]
nice wey nice

**Comment by laprakon** [comment]
Very very good

**Comment by cerkescnblt** [comment]
çok yavaş ilerliyor proje

**Reply by AyarNyein** [reply]
Really ?

**Comment by MrDino3813** [comment]
I personally will go for quality dapp even if it takes me months to build them. 

Not minding whatever barrier or time spent I will go for quality Dapp development

**Comment by OneNov0209** [comment]
Amazing

**Comment by Rdhlzrdy** [comment]
> Memungkinkan pengembangan kode rendah dan penerbitan koin kode nol, yang sangat mengurangi waktu dan hambatan untuk mengembangkan DApps menempatkan IBAX di atas yang sezaman

Good

**Comment by Rdhlzrdy** [comment]
Ok

**Comment by laprakon** [comment]
Nice

**Reply by AyarNyein** [reply]
really?

**Comment by ezioruan** [comment]
Nice

**Comment by zeland2323** [comment]
I don't believe in Dapps in the system yet, development is at a standstill.

**Comment by orjinim** [comment]
kesim

**Comment by aynurduzgun** [comment]
Bende yeni dapsslar gelebilir

**Comment by ezioruan** [comment]
Good job
